Sources/TuistCore/Graph/GraphTraverser.swift

```diff
@@ -474,6 +475,15 @@ public class GraphTraverser: GraphTraversing {
         })
     }
 
+    public func prebuiltDependencies(for rootDependency: GraphDependency) -> Set<GraphDependency> {
+        filterDependencies(
+            from: rootDependency,
+            test: { dep in
+                dep.isPrecompiled
```

Comment:
I'd just use anonymous parameter `$0.isPrecompiled` instead of `dep` which does not bring any real value here IMHO

Reply:
I changed it to keypath version 🚀